src/Microsoft.ServiceFabric.Actors/Runtime/ActorManager.cs

```diff
@@ -782,7 +782,7 @@ private void ArmGcTimer()
         {
             if (this.gcTimer != null)
             {
-                // if the host is closed, mark active actors for early collection rather than waiting till idleTimeout.
+                // Marks active actors for early collection rather than waiting till idleTimeout if the host is closed.
```

Comment:
This is a code comment telling the follow of the code, this change is not needed.